acestep/ui/gradio/interfaces/library_tab.py

```
@@ -0,0 +1,183 @@
+from typing import Any
+
+import gradio as gr
+
+from acestep.ui.gradio.interfaces.library_tab_handlers import (
+    delete,
+    do_refresh,
+    save_rating,
+    select_song,
+)
+
+_SORT_CHOICES = [("Newest First", "date"), ("Name A–Z", "name"), ("Top Rated", "rating")]
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)